dev/examples.el

```diff
@@ -1259,6 +1259,17 @@ new list."
     (funcall (-lambda (a b) (+ a b)) 1 2) => 3
     (funcall (-lambda (a (b c)) (+ a b c)) 1 (list 2 3)) => 6)
 
+  (defexamples -defun
+    (progn (-defun example/cdr ((_ . tail)) tail)
+           (example/cdr '(a . b))) => 'b
+    (progn (-defun example/car ((cur)) cur)
+           (example/car '(a . b))) => 'a
+    (progn (-defun example/add-cons ((a . b))
+             "Add the `car' and `cdr' of INPUT0."
+             (interactive (list (cons 1 2)))
+             (+ a b))
+           (command-execute #'example/add-cons)) => 3)
```

Comment:
Aren't examples limited to a single line? Have you tried regenerating the docs from this?

Reply:
I haven't yet tried regenerating the docs - I'll do that later. I see some two-line examples in `-setq`.